cmake/projects/Eigen/ci/matrix.json

```diff
@@ -0,0 +1,21 @@
+[
```

Comment:
It is okay for a toolchain to fail. You should remove this file.

Reply:
removed it